pandamium_datapack/data/pandamium/functions/tpa/accept_request.mcfunction

```diff
@@ -5,8 +5,8 @@ execute if score <can_accept> variable matches 0 run tellraw @p[tag=running_trig
 execute if score <can_accept> variable matches 1 run tellraw @p[tag=running_trigger] [{"text":"","color":"green"},{"text":"[TPA]","color":"blue"}," You ",{"text":"accepted","color":"aqua"}," ",[{"selector":"@s"},"'s"]," TPA request!"]
 execute if score <can_accept> variable matches 1 run tellraw @s [{"text":"","color":"green"},{"text":"[TPA]","color":"blue"}," ",{"selector":"@p[tag=running_trigger]"}," ",{"text":"accepted","color":"aqua"}," your TPA request!"]
 
+execute if score <can_accept> variable matches 1 run function pandamium:misc/teleport/anti_tp_exploit
 execute if score <can_accept> variable matches 1 run tp @s @p[tag=running_trigger]
 execute if score <can_accept> variable matches 1 if entity @s[gamemode=spectator] unless entity @s[x=-512,z=-512,dx=1024,dz=1024] unless score @s staff_perms matches 2.. run gamemode survival
-execute if score <can_accept> variable matches 1 run experience add @s 0
```

Comment:
anti_tp_exploit adds xp, but it does that before tping, doesn't it need to be done after?

Reply:
the order doesn't matter for some reason